dev-docs/source/Testing/VATES/VATESTEsting.rst

```diff
@@ -0,0 +1,61 @@
+.. _vates_testing:
```

Comment:
I will do a more thorough review when I'm back, but all references to "VATES" should be VSI. Vates was the original project name, which included visualisation and workspace extensions for multidimensional processing. VSI is vates-simple-interface, which we already refer to in all existing documentation. Personally I would prefer that this was called "n-d viewer" or something similar, but VSI is the best choice for now.